### PR TITLE
Updates all java files under security/* and test/security/a* -> test/security/m*

### PR DESCRIPTION
**Pull request by scrawfor99**
Follows #2823
Updates all java files under security/auth to security/filter.

### Check List
- [ ] ~New functionality includes testing~
- [x] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,7 +70,70 @@ apply plugin: 'opensearch.opensearchplugin'
...
 apply plugin: 'opensearch.rest-test'
 apply plugin: 'opensearch.testclusters'
-apply from: 'gradle/formatting.gradle'
+// apply from: 'gradle/formatting.gradle'
+
+spotless {
+    java {
+        // Normally this isn't necessary, but we have Java sources in
+        // non-standard places
+        target '**/com/amazon/dlic/**/*.java'
```
Comment (reta) [inline build.gradle:79]: Hm .. same as https://github.com/opensearch-project/OpenSearch/pull/7372 ?

Comment (scrawfor99) [reply build.gradle:79]: Hi @reta, sorry I am not sure I know what you are referencing here :/. I assume you are looking at the list of targets which I know is not the prettiest way of doing things. Unfortunately, I did not see a ton of options for creating a clearer description of which directories have the new formatting in place. 

Do you have a suggestion you would like me to follow?

Comment (reta) [reply build.gradle:79]: My sincere apologies, pasted wrong link here (not sure how), I meant this https://github.com/opensearch-project/security/pull/2823 - seems like doing the same changes ....

Comment (scrawfor99) [reply build.gradle:79]: Oh gotcha. Yeah, for whatever reason, there does not seem to be many changes within the target set here. I will rebase and expand the range to include more. It is hard to tell before rebasing what is actually new. Good catch.

Comment (scrawfor99) [reply build.gradle:79]: Now should have changes under `security/*` and `test/security/*`.

Looks like there were a lot of line break swaps in the tests...

**Review by reta — APPROVED** [review]

**Comment by scrawfor99** [comment]
@DarshitChanpura @RyanL1997 either of you have a chance to review this? :) Thank you

**Review by cwperks — APPROVED** [review]

**Pull request merged**